README.md

```diff
@@ -77,7 +77,7 @@ Run test upload using quetz-client. (For testing purposes, an API key is created
 
 ```bash
 export QUETZ_API_KEY=E_KaBFstCKI9hTdPM7DQq56GglRHf2HW7tQtq6si370
-quetz-client http://localhost:8000/api/channels/channel0 xtensor/linux-64/xtensor-0.16.1-0.tar.bz2 xtensor/osx-64/xtensor-0.16.1-0.tar.bz2
+quetz-client upload http://localhost:8000/api/channels/channel0 xtensor/linux-64/xtensor-0.16.1-0.tar.bz2 xtensor/osx-64/xtensor-0.16.1-0.tar.bz2
```

Comment:
thanks! 

I think we never released the version of the `quetz-client` where the `upload` subcommand is required. Do you think we can add a backwards-compatible "upload-without-subcommand" to the quetz-client?

Reply:
I didn't find a way using `argparse`.
we can set a default `subpasers`, but it's working only if there is no following arguments (which is not the case when uploading a package).
The only way I found is to add it manually in sys.argv, like : 

``` python
if len(sys.argv) > 1 and sys.argv[1] not in ALLOWED_SUBCOMMANDS:
    sys.argv.insert(1, "upload")
```

Reply:
Finally I implemented https://stackoverflow.com/a/46964652/15860586, the same idea as above but without changing `sys.argv`, only using `argparse`, which is better.